evelyn_chavez/python e/bike.py

```diff
@@ -0,0 +1,33 @@
+class Bike:
+    def __init__(self, price, max, miles):
```

Comment:
if you make it (self, price, max, miles = 0) then the miles have a default value of 0
On line 5 you can say self.miles = miles //this will make the miles 100, 80 and 60 respectively for bike1, bike2, bike3